backend/routes/user.js

```diff
@@ -0,0 +1,107 @@
+// const express = require('express');
```

Comment:
same idea with this file for the routes. Could have had this branched off and left this out of the pull request if this was not ready.